### PR TITLE
sysrepo-plugind improvements

### PR DESCRIPTION
**Pull request by Humblesaw**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -371,10 +371,10 @@ endif()
 check_symbol_exists(mkstemps "stdlib.h" SR_HAVE_MKSTEMPS)
 unset(CMAKE_REQUIRED_DEFINITIONS)
 
-# zip
-find_program(ZIP_BINARY "zip")
-if(NOT ZIP_BINARY)
-    message(FATAL_ERROR "zip binary was not found.")
+# tar
+find_program(TAR_BINARY "tar")
```
Comment (michalvasko) [inline CMakeLists.txt:375]: Add into dependencies in README.

File: src/executables/bin_common.h.in [hunk 2]
```diff
@@ -33,7 +33,7 @@
 #define SRPD_PLG_PATH "@SRPD_PLUGINS_PATH@"
 
 /** path to zip executable */
```
Comment (michalvasko) [inline src/executables/bin_common.h.in:35]: s/zip/tar

File: src/executables/sysrepo-plugind.c [hunk 3]
```diff
@@ -495,13 +495,13 @@ main(int argc, char **argv)
             if (get_plugins_dir(&plugins_dir)) {
                 goto cleanup;
             }
-            if (asprintf(&cmd, "/bin/cp -- \"%s\" %s", optarg, plugins_dir) == -1) {
+            if (asprintf(&cmd_arg, "-- \"%s\"", optarg) == -1) {
```
Comment (michalvasko) [inline src/executables/sysrepo-plugind.c:498]: `--` should probably be a separate argument.

**Review by michalvasko — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Humblesaw) [reply CMakeLists.txt:375]: Done

File: src/executables/bin_common.h.in (hunk 2 above)
Comment (Humblesaw) [reply src/executables/bin_common.h.in:35]: Done

File: src/executables/sysrepo-plugind.c (hunk 3 above)
Comment (Humblesaw) [reply src/executables/sysrepo-plugind.c:498]: Fixed

**Review by michalvasko — APPROVED** [review]

**Pull request merged**